kong/plugins/datadog/schema.lua

```diff
@@ -0,0 +1,8 @@
+return {
+  fields = {
+    host = {type = "string", default = "localhost"},
```

Comment:
Yes it runs locally as dadtadog agent.
On Dec 18, 2015 7:19 PM, "Thibault Charbonnier" <notifications@github.com>
wrote:

> In kong/plugins/datadog/schema.lua
> <https://github.com/Mashape/kong/pull/758#discussion_r48088242>:
>
> > @@ -0,0 +1,8 @@
> > +return {
> > +  fields = {
> > +    host = {type = "string", default = "localhost"},
>
> This should be required and probably should not have a default, since
> datadog does not usually runs locally right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Mashape/kong/pull/758/files#r48088242>.
>